src/Tools/FirstRunCertGenerator/test/CertificateManagerTests.cs

```diff
@@ -387,7 +387,7 @@ public void EnsureCreateHttpsCertificate_ReturnsExpiredCertificateIfVersionIsInc
         Output.WriteLine(creation.ToString());
         ListCertificates();
 
-        _manager.AspNetHttpsCertificateVersion = 2;
+        _manager.MinimumAspNetHttpsCertificateVersion = 2;
```

Comment:
I'd expect to see tests here for various permutations of minimum version vs actual version. For instance:

- Multiple certificates with version 1, 2, 3 and minimum set to 2.
- Downgrade scenario where there's a newer cert (say v3) and then the minimum is 2.